src/server/utils/onModuleLoad.js

```diff
@@ -64,6 +64,18 @@ function validateConfig(configValidators, config) {
 
 export const CONFIGURATION_KEY = 'appConfig';
 
+export function cspCheck(csp) {
+  if (!csp && !process.env.ONE_DANGER_DISABLE_CSP) {
+    throw new Error('Root module must provide a valid content security policy. If you would like to bypass this, set the ONE_DANGER_DISABLE_CSP enviornment variable to *, and NODE_ENV to development.');
+  }
+  if (process.env.ONE_DANGER_DISABLE_CSP && process.env.NODE_ENV !== 'development') {
```

Comment:
Assert on the value of the env var rather than the presence

Do this validation in [`runTime.js`](https://github.com/americanexpress/one-app/blob/main/src/server/config/env/runTime.js)

Reply:
are you talking about doing something like 
```js 
{
    name: 'ONE_DANGER_DISABLE_CSP',
    normalize: (input) => input && input.toLowerCase(),
    valid: ['true', 'false'],
    defaultValue: 'false',
  },
  ```

Reply:
You can normalize to a bool and check for `NODE_ENV=development`  in `validate`. This lets the app fail to start before loading modules